docs/pages/develop/authentication.mdx

```diff
@@ -1,67 +1,288 @@
+
+<Collapsible summary="Biometrics">
+
+Biometrics like Face ID and Touch ID can be used to unlock the app or confirm identity after a valid session is established. These are not authentication methods on their own, but act as a local gate that makes re-authentication faster and more secure.
+
+React Native provides access to biometric APIs through libraries like [expo-local-authentication](/versions/latest/sdk/local-authentication) or [react-native-biometrics](https://github.com/SelfLender/react-native-biometrics).
+
+</Collapsible>
+
+<Collapsible summary="Passkeys">
+
+[Passkeys](https://safety.google/authentication/passkey) are a new, passwordless way to log in to apps and websites. Backed by Apple, Google, and Microsoft, they use platform-level cryptography and biometrics to authenticate users without passwords.
+
+Passkeys offer a seamless and secure experience, but they require a user to already be authenticated before registering one. They also require extra configuration if you're not using a provider that handles them for you.
+
+- React Native passkey support: [react-native-passkeys](https://github.com/peterferguson/react-native-passkeys)
+- Native passkey support with Clerk: [Clerk Passkeys for Expo](https://clerk.com/docs/references/expo/passkeys)
+
+</Collapsible>
+
+## Recommendations
+
+This guide covers a lot of ground, from basic email and password flows to fully custom OAuth implementations, session management, and modern methods like biometrics and passkeys. Not all of these need to be implemented at once.
+
+In many cases, starting simple is the best approach. Shipping your app with something like email authentication using a magic link or one-time passcode is often more than enough to get through the App Store review process and start collecting feedback from real users.
+
+That said, if you're building an app where you expect high traffic from day one or need to support sign-in across platforms with minimal friction, investing in a more complete authentication flow early on can make a big difference. It can help improve user onboarding, trust, and retention right from the start.
+
+Modern solutions like OAuth, biometrics, and passkeys are not required, but they can be excellent additions once your core system is in place. The key is to build authentication that fits your current needs, while staying flexible enough to grow with your product.
+
+It's perfectly okay to start with a third-party service and refine your approach as your app matures.
+
 <BoxLink
   title="Authentication with OAuth or OpenID providers"
   description="Learn how to utilize the expo-auth-session library to implement authentication with OAuth or OpenID providers."
   href="/guides/authentication"
   Icon={BookOpen02Icon}
 />
+
+<BoxLink
+  title="Google Sign-In with Expo Router API Routes"
+  description="Learn how to implement Google Sign-In with Expo Router API Routes"
+  href="/guides/google-authentication"
```

Comment:
It seems the link to the guide and the title, description do not match each other.